src/geophires_x/AGSWellBores.py

```diff
@@ -1,11 +1,13 @@
-# copyright, 2023, Malcolm I Ross
-# except for code from Wanju Yuan based on: "Closed-Loop Geothermal Energy Recovery from Deep High Enthalpy Systems"
-# ref: Yuan, Wanju, et al. "Closed-loop geothermal energy recovery from deep high enthalpy systems."
-# Renewable Energy 177 (2021): 976-991.
-# and CLGScode from Koenraad
-# ref Beckers, Koenraad, et al. Tabulated Database of Closed-Loop Geothermal Systems Performance for Cloud-Based
-# Technical and Economic Modeling of Heat Production and Electricity Generation. No. NREL/CP-5700-84979.
-# National Renewable Energy Lab.(NREL), Golden, CO (United States), 2023.
+"""
+ contains code from Wanju Yuan based on: "Closed-Loop Geothermal Energy Recovery from Deep High Enthalpy Systems"
+ ref: Yuan, Wanju, et al. "Closed-loop geothermal energy recovery from deep high enthalpy systems."
+ Renewable Energy 177 (2021): 976-991.
+ and CLGScode from Koenraad
+ ref Beckers, Koenraad, et al. Tabulated Database of Closed-Loop Geothermal Systems Performance for Cloud-Based
+ Technical and Economic Modeling of Heat Production and Electricity Generation. No. NREL/CP-5700-84979.
+ National Renewable Energy Lab.(NREL), Golden, CO (United States), 2023.
+ :doc-author: Malcolm Ross
```

Comment:
Suggest removing these `doc-author` entries - they're redundant with git history (and become inaccurate when someone else touches docstrings)

Reply:
Addressed by https://github.com/NREL/python-geophires-x/pull/41/commits/e3c6c4c79425976085a7d939e4e00a308146a1b4, resolving